fluent-react/src/context.ts

```diff
@@ -1,4 +1,5 @@
 import { createContext } from "react";
 import { ReactLocalization } from "./localization";
 
-export let FluentContext = createContext(new ReactLocalization([], null));
+export let FluentContext =
+  createContext(null) as React.Context<ReactLocalization | null>;
```

Comment:
The addition of `ReactLocalization | null` means that the rest of the code will now need null checks.